src/x509.c

```diff
@@ -8621,6 +8621,8 @@ WOLF_STACK_OF(WOLFSSL_X509)* wolfSSL_X509_chain_up_ref(
             return WOLFSSL_FAILURE;
         }
 
+        cert->extBasicConstSet = x509->basicConstSet;
```

Comment:
This seems like it belongs in `CertFromX509` rather than this function.

Reply:
Will move it.